lib/aca_entities/atp/functions/vlp_document_hash_builder.rb

```diff
@@ -91,6 +93,10 @@ def handle_identification_category(vlp_document, document_person_id)
             vlp_document[:sevis_id] = document_person_id[:identification_id]
           when 'Visa Number'
             vlp_document[:visa_number] = document_person_id[:identification_id]
+          when /certificate\s*of\s*citizenship/i
```

Comment:
Are we expecting the spacing and capitalization to be variable here?  I wonder if we should be making these cases so flexible.  We don't have this in place for most of the other case strings.  

Not a major concern, but I feel like the inconsistency is a bit confusing.  Also, it could be misleading that we are matching this way if the reality is that we always expect the incoming payload to contain `Certificate Of Citizenship` or `Naturalization Certificate Number`.

Reply:
Ok, that is very helpful context.  Thank you!  Could be worth a comment in the code here explaining that we're using a regex in the cases where we've seen variability in what the external service sends.